Final 0.6.4 Release Notes (#822)
napari/docs: docs/release/release_0_6_4.md

### Run scripts with napari from the command line

As a follow-up to the ability to drag-n-drop scripts into the napari window from 0.6.3, you can now run scripts directly from the command line using the `napari` command and the path to the script ([#8185](https://github.com/napari/napari/pull/8185) and [#8187](https://github.com/napari/napari/pull/8187)).

> `napari examples/magic_immage_arithmetic.py` will open a local napari and run the local script.

Reviewer: ```suggestion
To open a local napari and run a local script, enter: `napari examples/magic_immage_arithmetic.py`.
```

You can also run scripts from a remote location ([#8208](https://github.com/napari/napari/pull/8208)), including Github, Gist, Gitlab, and the napari gallery.

> For example, you can run `napari https://github.com/napari/napari/blob/main/examples/grid_mode.py`.

Reviewer: ```suggestion
To run a remote script, for example, enter: `napari https://github.com/napari/napari/blob/main/examples/grid_mode.py`.
```
